doc/man3/OSSL_PARAM_build.pod

```diff
@@ -0,0 +1,191 @@
+=pod
+
+=head1 NAME
+
+OSSL_PARAM_build, OSSL_PARAM_build_free,
+OSSL_PARAM_BUILD_BN, OSSL_PARAM_BUILD_double,
+OSSL_PARAM_BUILD_int, OSSL_PARAM_BUILD_int32,
+OSSL_PARAM_BUILD_int64, OSSL_PARAM_BUILD_long,
+OSSL_PARAM_BUILD_size_t, OSSL_PARAM_BUILD_uint,
+OSSL_PARAM_BUILD_uint32, OSSL_PARAM_BUILD_uint64,
+OSSL_PARAM_BUILD_ulong, OSSL_PARAM_BUILD_BN,
+OSSL_PARAM_BUILD_utf8_string,
+OSSL_PARAM_BUILD_octet_string,
+- OSSL_PARAM array helpers
+
+=head1 SYNOPSIS
+
+=for comment generic
+
+ #include <openssl/params.h>
+
+ #define OSSL_PARAM_BUILD_TYPE(key, address)
+ #define OSSL_PARAM_BUILD_utf8_string(key, address, size)
+ #define OSSL_PARAM_BUILD_octet_string(key, address, size)
+ #define OSSL_PARAM_BUILD_BN(key, address, size)
+
+ OSSL_PARAM *OSSL_PARAM_build(const char *name, ..., NULL);
+ void OSSL_PARAM_build_free(OSSL_PARAM *params);
+
+=head1 DESCRIPTION
+
+A collection of utility functions that simplify building arrays of
+OSSL_PARAM structures.  The following B<TYPE> names are supported:
+
+=over 1
+
+=item *
+
+double
+
+=item *
+
+int
+
+=item *
+
+int32 (int32_t)
+
+=item *
+
+int64 (int64_t)
+
+=item *
+
+long int (long)
```

Comment:
why "long int" and not just "long" ?
because "long int" isn't really a supported value for the TYPE is it. :)
And flip the lines so the TYPE appears first, then the native C type in parens.